nextstrain/cli/command/view.py

```diff
@@ -37,17 +37,45 @@
 .. _narrative: https://docs.nextstrain.org/page/reference/glossary.html#term-narrative
 """
 
+import multiprocessing
 import re
+import webbrowser
+from os import environ
 from pathlib import Path
 from socket import getaddrinfo, AddressFamily, SocketKind, AF_INET, AF_INET6, IPPROTO_TCP
+from time import sleep
 from typing import Iterable, NamedTuple, Tuple, Union
 from .. import runner
-from ..argparse import add_extended_help_flags, SUPPRESS
+from ..argparse import add_extended_help_flags, SUPPRESS, SKIP_AUTO_DEFAULT_IN_HELP
 from ..runner import docker, ambient, conda
 from ..util import colored, remove_suffix, warn
 from ..volume import store_volume
 
 
+# Always use the "spawn" start method which is available on all platforms,
+# instead of using the platform-dependent default (e.g. "spawn" on Windows and
+# macOS but "fork" on other Unixes).  The semantics and behaviour of forking is
+# so different from spawning re: shared variables and program state that it's
+# easier to use the same method everywhere (even if forking is nicer than
+# spawning).
+mp_spawn = multiprocessing.get_context("spawn")
+Process = mp_spawn.Process
+ProcessError = mp_spawn.ProcessError
+
+
+# Avoid text-mode browsers
+TERM = environ.pop("TERM", None)
```

Comment:
I went down a tiny rabbit hole of what `TERM` means. Finally found it being used in the [webbrowser code for registering browers](https://github.com/python/cpython/blob/abbe4482ab914d6da8b1678ad46174cb875ed7f6/Lib/webbrowser.py#L568) 😄

Reply:
Also learned that webbrowser checks `BROWSER` for user specified browser. This works with `nextstrain view` to use a specific browser rather than the default!